avocado/core/job.py

```diff
@@ -150,11 +150,13 @@ def _remove_job_results(self):
 
     def _make_test_loader(self):
         for key in self.args.__dict__.keys():
-            if key.endswith('_loader'):
-                loader_class = getattr(self.args, key)
+            loader_class = getattr(self.args, key)
+            try:
                 if issubclass(loader_class, loader.TestLoader):
                     loader_plugin = loader_class(self)
                     self.test_loader.add_loader_plugin(loader_plugin)
+            except TypeError:
```

Comment:
Oh, thanks for the info, I never thought it would that obvious. I thought that `issubclass` behavior was limited to returning `True` or `False`.